### DISCUSSION TITLE
cycle at stratify err

### DISCUSSION THREAD

**Discussion by Ramkumar-r7** [d3/d3#3798] [Q&A]
I was using d3 org chart and it was working fine in the initial stage but after that this stratify error came i dont know what actual code is causing this error
![image](https://github.com/d3/d3-hierarchy/assets/115092135/df0c6115-ce9e-42b4-b17a-43b411684907)

**Comment by Fil** [comment]
The error is saying that there is a cycle in your data, where x is a parent of y and y a parent of x. Sometimes this is caused by an empty line which is interpreted as ("" is a parent of itself), or something similar.
